exercises/01-react-stateful/src/modules/root/components/root.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+import { UsersList } from 'modules/users/components/users-list'
+
+import { Header } from './header'
+
+export const Root: React.FC = () => (
```

Comment:
Same here, `React.FC` is not necessary.

Reply:
considering we want to have an explicit return type in functions, I'm fine keeping it here as long as it's consistent

Reply:
Leaving as it is